client/blocks/plan-thank-you-card/docs/example.jsx

```diff
@@ -24,7 +24,11 @@ function PlanThankYouCardExample( { primarySiteId } ) {
 }
 
 const ConnectedPlanThankYouCard = connect( ( state ) => {
-	const primarySiteId = get( getCurrentUser( state ), 'primary_blog' );
```

Comment:
I agree with adding the default arg, and removing the if block below.

I would suggest `null` or `-1` over `false` as the default since I would not expect a boolean value from the variable name.
```javascript
const primarySiteId = get( getCurrentUser( state ), 'primary_blog', null );
```